pages/runtime/update-locking.md

```diff
@@ -1,8 +1,8 @@
 ## Update locking
 
-Locking updates means that the Supervisor will not be able to kill your application. This is meant to be used at critical sections of your code where you don't want to be interrupted, or to control that updates are only installed at certain times.
+Locking updates means that the Supervisor will not be able to kill your application. This is meant to be used at critical sections of your code where you don't want to be interrupted, or to ensure that updates are only installed at certain times.
```

Comment:
I think this should be `will not kill your application` because it can do it, and it will if a lock override is set.